fritz.defaults.yaml

```diff
@@ -51,6 +51,8 @@ skyportal:
 
     treasuremap_endpoint: http://treasuremap.space
 
+    tns_endpoint: https://sandbox.wis-tns.org
```

Comment:
Is it possible that this was deliberately removed from the fritz config (used in staging) to prevent accidentally posting things to TNS?

Reply:
I guess. Is that going to help make some error go away?

Reply:
yeah it was making it impossible for docs to build and such